src/vivarium_gates_iv_iron/data/utilities.py

```diff
@@ -0,0 +1,239 @@
+from itertools import product
```

Comment:
Is this all just copied from CIFF? Is there anything I should be pat special attention to?

Reply:
All CIFF copypasta.